src/types/branch-comparison.ts

```diff
@@ -0,0 +1,29 @@
+export interface BranchComparison {
+  /**
+   * The comparative status of the head branch to the base branch.
+   *
+   * Example: "diverged" | "ahead" | "behind" | "identical"
+   */
+  branchStatus: string
```

Comment:
Worth making this an enum?

Reply:
yep, its on the list with https://github.com/crs-k/stale-branches/blob/ebc6cfd880f2b9396242d78c1a8f7c60a01ecb55/src/types/inputs.ts#L37 and https://github.com/crs-k/stale-branches/blob/ebc6cfd880f2b9396242d78c1a8f7c60a01ecb55/src/types/inputs.ts#L21

Reply:
i actually don't know if i need to do the yml boolean check since it'll break during validation. the response type for the commit comparisons also shouldn't change, but i'll do both when i work through learning a clean way to use enums as keys for a type's field.